lib/rules/nesting-depth.js

```diff
@@ -2,6 +2,15 @@
 
 var helpers = require('../helpers');
 
+var lodash = require('lodash');
+
+var compiledCSSDoesNotCreateDepth = function (selector) {
+  return lodash.isArray(selector.content) &&
```

Comment:
You could just include the lodash.isarray package if you require it, no point writing and testing our own.